app/_data/docs_nav_gateway_3.4.x.yml

```diff
@@ -164,6 +164,7 @@ items:
             url: /licenses/examples
           - text: Monitor Licenses Usage
             url: /licenses/report
+            # src: /gateway/licenses/report-v2
```

Comment:
This one is commented out so that we know to add it when the patch release goes out.